tests/source/issue-5359/code_comment_width_greater_max_width.rs

```diff
@@ -0,0 +1,17 @@
+// rustfmt-max_width: 50
```

Comment:
These tests look good! However, we actually put tests for config option in a special directory, named `configs` under the source and target dirs

Please move these when you get a chance, and with that move you may be able to rename the files (though don't feel obligated to do so)
https://github.com/rust-lang/rustfmt/tree/master/tests/source/configs